### ISSUE TITLE
Advices for intereaction enhancement

### ISSUE THREAD

**Issue by AiurArtanis** [Aidurber/tag-page-preview#12]
@Aidurber 

## Advices

1. customisation for showing the preview page of some tag.
sometimes i want change the text of one tag and the preview page of this tag shows. maybe it'll be a safer way to call the page that click the tag with `Shift` or `Alt`/`Option`

2. style customisation for the preview page.
such as page title text, notes list style...

3. note preview in preview page of tags.
support the note preview with pressing `Ctrl`/`Command` in the page

## Environment

OS:  win 10
Ob: v0.13.23
Tag-Page-Preview: 0.0.12

## Last

what a nice plugin😁 and there's always room for improvement

**Comment by fabionolasco** [comment]
I agree. These three feature requests are great and were listed in the same order of importance that I would've listed them.

**Comment by reorx** [comment]
Note preview is a must-have, I want the context of the tag shown under the file link.

I'm considering forking this plugin if Aidurber has no interest in taking it further.
